llvm/lib/MC/MCSymbol.cpp

```diff
@@ -84,7 +84,22 @@ void MCSymbol::print(raw_ostream &OS, const MCAsmInfo *MAI) const {
 }
 
 #if !defined(NDEBUG) || defined(LLVM_ENABLE_DUMP)
-LLVM_DUMP_METHOD void MCSymbol::dump() const {
-  dbgs() << *this;
-}
+LLVM_DUMP_METHOD void MCSymbol::dump() const { dbgs() << *this; }
 #endif
+
+bool llvm::isRangeRelaxable(const MCSymbol *Begin, const MCSymbol *End) {
```

Comment:
Please can we not re-implement `absoluteSymbolDiff` from `MCObjectStreamer.cpp` - given the logic there is already quite complex, maybe we can re-use that function rather than missing this one in a future update?